### ISSUE TITLE
Scroll View

### ISSUE THREAD

**Issue by dmikey** [dmikey/syr#48]

**Comment by gabrielcsapo** [comment]
is this a duplicate of https://github.com/dmikey/syr/issues/11?

**Comment by dmikey** [comment]
@gabrielcsapo nope, this one is for web. The other one was missing a label, but both are in separate projects.

**Comment by gabrielcsapo** [comment]
Okay just checking @dmikey
